index.html

```diff
@@ -20,377 +20,272 @@
+        <div class="frame2">
+          <div class="top-frame2">
+            <div class="rectangle"></div>
+            <div class="framework">
+              <h4>Frameworks</h4>
+            </div>
           </div>
-        </div>
-        <div class="inner-frame">
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        </div>
-      </div>
-      <div class="framework">
-        <div class="sub-framework">
-          <div class="square"></div>
-          <div class="frame">
-            <h4>Frameworks</h4>
+          <div class="frame-content">
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RsPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
           </div>
+
         </div>
-        <div class="inner-framework">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-      </div>
-      <div class="main-skill">
-        <div class="sub-skills">
-          <div class="circle"></div>
-          <div class="skills">
-            <h4>Skills</h4>
+
+        <div class="frame3">
+          <div class="top-frame3">
+            <div class="circle"></div>
+            <div class="skills">
+              <h4>skills</h4>
+            </div>
           </div>
-        </div>
-        <div class="inner-skills">
-          <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+          <div class="skill-content">
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+
         </div>
       </div>
-      
-
-      
-      
-
     </section>
 
     <section class="contact">
       <div class="contact-text">
-        <p>
-          I'm always interested in hearing about new projects, 
+        <p>I'm always interested in hearing about new projects, 
           so if you'd like to chat please get in touch.
         </p>
       </div>
+      <form action="https://formspree.io/f/mnqreddp" method="post" class="form">
+        <div class="form-row">
+          <input type="text" class="form-text" maxlength="30" placeholder="first name" required>
+        <input type="text"  class="form-text" maxlength="30" placeholder="last name" required>
+        </div>
+        <div class="form-mail">
+          <input type="email"  class="contact-form-email" placeholder="Email address" required>
+        </div>
+          <textarea name="contact"  class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
```

Comment:
- [ ] [**OPTIONAL**] Kindly make sure your design stick as much as possible to the figma design :+1:, because it's an crucial part of the[ project requirements.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.)

Your Design
![Screenshot from 2022-12-10 04-08-40](https://user-images.githubusercontent.com/80458199/206824168-34b0a7da-c942-4f65-8445-dcf7b09d5a4a.png)

Figma design

![Screenshot from 2022-12-10 04-09-53](https://user-images.githubusercontent.com/80458199/206824177-a3b36064-85f1-4a43-a6e2-d49983bf9203.png)